.circleci/config.yml

```diff
@@ -96,6 +101,29 @@ jobs:
           path: ios/PurchasesHybridCommon/test_output
           destination: scan-output
 
+  test-xcode-13-2:
+    executor: ios-executor-xcode-13-2
+    steps:
+      - checkout
+      - run:
+          name: Install pods
+          command: pod install --repo-update
+          working_directory: ios/PurchasesHybridCommon
+      - run:
+          name: Run ios tests
+          command: fastlane scan
+          working_directory: ios/PurchasesHybridCommon
+          environment:
+            SCAN_SCHEME: PurchasesHybridCommon
+      - run:
+          name: Run pod lint tests
+          command: pod lib lint --fail-fast
+      - store_test_results:
+          path: ios/PurchasesHybridCommon/test_output
+      - store_artifacts:
+          path: ios/PurchasesHybridCommon/test_output
+          destination: scan-output
```

Comment:
Hmm it would be great if we could share all the steps and just change the executor, but I don't know if CircleCI supports that... This is ok though.

Reply:
Yeah done.